Hardware/Hardware/Printer/AirPrintReceipt/AirPrintReceiptPrinterService.swift

```diff
@@ -38,3 +44,27 @@ public final class AirPrintReceiptPrinterService: PrinterService {
         }
     }
 }
+
+extension AirPrintReceiptPrinterService: UIPrintInteractionControllerDelegate {
+    public func printInteractionController(_ printInteractionController: UIPrintInteractionController, choosePaper paperList: [UIPrintPaper]) -> UIPrintPaper {
+        let pageSize = self.receiptContent?.preferredPageSizeForPrinting ?? AirPrintReceiptPrinterService.defaultReceiptPageSize
+        return UIPrintPaper.bestPaper(forPageSize: pageSize, withPapersFrom: paperList)
+    }
+
+    public func printInteractionController(_ printInteractionController: UIPrintInteractionController, cutLengthFor paper: UIPrintPaper) -> CGFloat {
+        let pageSize = self.receiptContent?.preferredPageSizeForPrinting ?? AirPrintReceiptPrinterService.defaultReceiptPageSize
+        return pageSize.height + AirPrintReceiptPrinterService.defaultRollCutterMargin
+    }
+}
+
+extension AirPrintReceiptPrinterService {
+    static let pointsPerInch: Int = 72
+
+    /// Default margin for roll cutter.
+    ///
+    static let defaultRollCutterMargin: CGFloat = CGFloat(1 * pointsPerInch)
+
+    /// Default size of a page for a receipt in points.
+    ///
+    static let defaultReceiptPageSize: CGSize = CGSize(width: 8 * pointsPerInch, height: 11 * pointsPerInch)
```

Comment:
Why the default 8" here and 4" on `ReceiptContent`? Is this a default for non-roll printers?

Reply:
Fixed in e17abf9af525d46d99c305995b631f3175435cff